.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,100 @@
+name: Docker Image Build
+
+env:
+  IMAGE_NAME: pegasystems/pega-ready
+  VERSION: 2.1.0-${GITHUB_REPOSITORY}-${GITHUB_SHA}
```

Comment:
We've bumped the major version to 3.0.build# when we updated the base image to Ubuntu 20.04
Do GitHub actions provide build#'s?  Doesn't look like it.  So we should consider how we increment the patch as we apply the OS updates